docs/development/style-guide.rst

```diff
@@ -208,49 +229,101 @@ Example
-  * To test if your version of ``astropy`` is running correctly…
+The present tense. For example:
+* In the following section, we are going to make a plot...
+* To test if your version of ``astropy`` is running correctly...
 
-* The first-person inclusive plural. For example:
-  * We did this the long way, but next let’s try the short way…
+The first-person inclusive plural. For example:
+* We did this the long way, but next we can try it the short way...
 
-* Use the generic pronoun “you” instead of “one.” For example:
-  * You can access any of the attributes on a frame by...
+Use the generic pronoun “you” instead of “one.” For example:
+* You can access any of the attributes on a frame by...
 
 Always avoid extraneous or belittling words such as “obviously,” “easily,”
 “simply,” “just,” or “straightforward.” Avoid extraneous phrases like, “we just
 have to do one more thing.”
 
+Avoid words or phrasing that create worry in the mind of the reader, such as an
+overuse of "Warnings." Instead, use positive language that establishes
+confidence in the skills being learned, such as "As a best practice..." or "One
+recommended way to..." or "An important note to remember is..."
+
 Documentation vs. Tutorials vs. Guides
 --------------------------------------
 
 Documentation
-~~~~~~~~~~~~~
+^^^^^^^^^^^^^
 Tone: academic and slightly more formal.
 
-* Use up style capitalization in section headings.
+* Use title case capitalization in section headings.
 * Do not use contractions.
 
 Tutorials
-~~~~~~~~~
+^^^^^^^^^
 Tone: academic but less formal and more friendly.
 
-* Use up style capitalization in introductory/template headings, switch to down
-  style capitalization for learning/example section headings.
+* Use title case capitalization in introductory/template headings, switch to
+  sentence case capitalization for learning/example section headings.
 * Section headings should use the imperative mood to form a command or request
   (e.g., “Download the data”).
 * Contractions can be used as long as the tense is clear.
 
 Guides
-~~~~~~
+^^^^^^
 Tone: academic but less formal and more friendly.
 
-* Use up style capitalization in introductory/template headings, switch to down
-  style capitalization for learning/example section headings.
+* Use title case capitalization in introductory/template headings, switch to
+  sentence case capitalization for learning/example section headings.
 * Contractions can be used as long as the tense is clear.
+
+Formatting Guidelines
+=====================
+
+Astropy documentation is written in reStructuredText using the Sphinx
+documentation generator. When formatting the different sections of your
+documentation files, please follow these guidelines to maintain consistency in
+section heading hierarchy across Astropy's RST files.
+
+Section headings in reStructuredText files are created by underlining (and
+optionally overlining) the section title with a punctuation character at least
```

Comment:
Remove "at least" - we always seem to make it equally long, which looks (I think) best.